src/linear_algebra/finite_dimensional.lean

```diff
@@ -828,6 +828,18 @@ by { unfold findim, simp [dim_top] }
 
 end top
 
+lemma finite_dimensional.is_basis_of_findim_eq_zero [finite_dimensional K V]
```

Comment:
You can probably drop `finite_dimensional` from the name here.

Reply:
Also, do we already have this of `dim K V = 0`?

Reply:
I do not think so (at least I can't find it in `finite_dimensional` nor `basis`). Should I add it?

Reply:
Yeah, I think that would be a good idea.